.github/workflows/actions.yml

```diff
@@ -793,3 +793,42 @@ jobs:
         with:
           name: binaries-${{ github.job }}
           path: pfx/**/*
+
+  android:
+    needs: [ what-to-make ]
+    runs-on: ubuntu-22.04
+    env:
+      VCPKG_DEFAULT_TRIPLET: x64-android
```

Comment:
Given that we're building for Android, should this be `arm64` (and not `x64`) instead? I see that build.gradle also has `x86_64` though. Feels like arm64 build will be closer to what people actually use in the end, but then I'm not sure if we'll be able to run unit tests easily (BTW, can we do this and/or does it make sense here? I'm okay with not running them and just testing that build works for now)

Reply:
You are right, `arm64` will be the main target for android. I switched to `arm64` for CI building.
I guess running tests would require starting an android emulator ? It would be great having them running, I guess this could be done in a future PR ?